Libraries/Components/ScrollResponder.js

```diff
@@ -471,6 +472,15 @@ var ScrollResponderMixin = {
    * @platform ios
    */
   scrollResponderFlashScrollIndicators: function() {
+    if (Platform.OS === 'android') {
```

Comment:
I think ViewManagers as modules were just never implemented on Android. I was mostly basing this comment on other methods like scrollTo uses only dispatchViewManagerCommand even on iOS. On iOS react nativr is already built from source so you could test by either running the RNTester xcodeproj or changing code under node_modules in an app.